_notices/rsn0028.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 28 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Support for Rocky 8.5 in v22.08"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: green
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.08+"
+notice_created: 2022-06-08
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-06-08
+---
+
+## Overview
+
+Development effort will begin supporting `Rocky 8.5` in `v22.08` nightlies. `Rocky 8.5` will replace `CentOS 7` and `CentOS 8` OS distributions for future RAPIDS releases.  Support will be experimental for `v22.08`. GA support for `Rocky 8.5` is expected in `v22.08`
```

Comment:
I think Rocky replaces Centos 8 but not Centos 7, right @raydouglass ?

Reply:
In our POR for this change, we didn't mention dropping centos 7 support either, but maybe I missed an updated version?

Reply:
While i can understand that it may be a technically inaccurate statement in what it actually replaces, I don't think we did the CentOS 7 dep, so i was bundling it to end it as well and reduce later confusion.  Would better wording be "supersede" instead of "replace"?

Reply:
resolved in latest commit. thanks @JohnZed !